packages/bus-redis/src/redis-transport.ts

```diff
@@ -166,10 +158,13 @@ export class RedisTransport implements Transport<RedisMessage> {
       stickyAttributes: messageOptions.stickyAttributes
     }
     this.logger.debug('Sending message to Redis', {payload})
-    await this.queue.add(message.$name, payload, {
-      jobId: uuid.v4(),
-      attempts: this.maxRetries,
-      removeOnComplete: true
-    })
+    const serializedPayload = JSON.stringify(payload)
+    const queues = await this.getQueuesSubscribedToMessage(message)
+    await Promise.all(queues.map(async queueName => {
```

Comment:
all settled might be a better option so that a failure doesn't affect others. 

```suggestion
    await Promise.allSettled(queues.map(async queueName => {
```

might need to have a think what a failure means in this case. would we want to throw an error, retry and therefore double-send to the other queues? 

maybe here we just want to retry the send to the failed subscriber a couple of times, log an error and give up if it's undeliverable.

Reply:
Good call, I've added a recursive retry approach - should be tail call optimised as the retry call is in the catch block.

Reply:
@andrew have added the allSettled approach - this is ready for another look :pray: